LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Center for Computation and Visualization, Brown University
```

Comment:
@mirestrepo should this be CCV or RIDDC?

Reply:
Rhode Island Data Discovery Center, right?

Reply:
I changed it to "Copyright (c) 2021 Rhode Island Data Discovery Center"

Will wait to merge until I get confirmation that RIDDC should own the copyright for the project

Reply:
I asked Jeff and he asked if we could put both RIDDC and Brown University